test/integration/test_enterprise_management_v1.py

```diff
@@ -15,389 +14,265 @@
+        )
+
+        assert list_account_groups_response.get_status_code() == 200
+        list_account_groups_response = list_account_groups_response.get_result()
+        assert list_account_groups_response is not None
+
+    @needscredentials
+    def test_list_account_groups_with_pagination(self):
+
+        results_count = 0
+        doc_id = None
+
+        while True:
+            list_account_groups_response = self.enterprise_management_service.list_account_groups(
+                enterprise_id=self.enterprise_id,
+                limit=result_per_page,
+                next_docid=doc_id,
+            )
+
+            assert list_account_groups_response.get_status_code() == 200
+            list_account_groups_response = list_account_groups_response.get_result()
+            assert list_account_groups_response is not None
+
+            results_count += 1
+
+            next_url = list_account_groups_response.get('next_url')
+            if next_url is None:
+                break
+
+            doc_id = get_query_param(next_url, 'next_docid')
+
+        print(f'\nlist_account_groups() returned a total of {results_count} account groups.')
+
+    @needscredentials
+    def test_get_account_group(self):
+        assert first_example_account_group_id is not None
+
+        get_account_group_response = self.enterprise_management_service.get_account_group(
+            account_group_id=first_example_account_group_id,
+        )
+
+        assert get_account_group_response.get_status_code() == 200
+        account_group = get_account_group_response.get_result()
+        assert account_group is not None
+
+    @needscredentials
+    def test_update_account_group(self):
+        assert first_example_account_group_id is not None
+
+        update_account_group_response = self.enterprise_management_service.update_account_group(
+            account_group_id=first_example_account_group_id,
+            name=first_updated_example_account_group_name,
+            primary_contact_iam_id=self.account_iam_id,
+        )
+
+        assert update_account_group_response.get_status_code() == 204
+
+    @needscredentials
+    def test_create_account(self):
+        global example_account_id
+
+        assert first_example_account_group_id is not None
+
+        parent = 'crn:v1:bluemix:public:enterprise::a/' + self.account_id + '::account-group:' + first_example_account_group_id
+
+        create_account_response = self.enterprise_management_service.create_account(
+            parent=parent,
+            name=example_account_name,
+            owner_iam_id=self.account_iam_id,
+        )
+
+        assert create_account_response.get_status_code() == 202
+        create_account_response = create_account_response.get_result()
+        assert create_account_response is not None
+
+        example_account_id = create_account_response.get('account_id')
+
+    @needscredentials
+    def test_list_accounts(self):
```

Comment:
for list operations, I think testing the operation with pagination is sufficient since the "without pagination" scenario is a subset of the "with pagination" scenario.